kavm/src/kavm/__main__.py

```diff
@@ -290,7 +295,13 @@ def create_argument_parser() -> ArgumentParser:
         '--teal-programs-parser',
         dest='teal_programs_parser',
         type=file_path,
-        help='Path to the executable to parse .teals files containing TealPrograms terms',
+        help='Path to the executable to parse TealProgramStore objects containing TealPrograms terms',
+    )
```

Comment:
The `--teal-programs-parser` option is obsolete now, since you're using the sort `Map[Id, TealInputPgm]`.

Reply:
I removed this option.